docs/modules/ROOT/pages/references/architecture/metering-data-flow.adoc

```diff
@@ -257,6 +258,10 @@ For example, the ingestion job may need to sum all records belonging to a single
 | appuio_cloud_memory
 | appuio_cloud_memory_requested
 
+| is_subquery
```

Comment:
Now we reach the stage, where the naked SQL API might no longer be sufficient. If not taken care, values in this column and the content in the mapping table might drift apart. So I would prefer to not add it in the first place.

I assume, the reason for adding it is, it makes querying a lot easier (fewer joins). Are there any others?